lib/_stream_writable.js

```diff
@@ -238,6 +238,9 @@ function Writable(options) {
   }
 
   Stream.call(this);
+
+  // To identify this unique stream object
+  // this.streamId = Symbol('writable');
```

Comment:
Is this a leftover?

Reply:
Yes, I forgot to remove them... I'll clean them up.